.github/workflows/ci.yml

```diff
@@ -29,6 +24,39 @@ jobs:
       - name: Build
         run: npm run build
 
+  test:
```

Comment:
it looks like this is the same as build

Reply:
For me (on windows) this is not the case. Either `yarn build` or `npm run build`.

Reply:
the existing flow does a build & test, I think I'd keep it that way